src/main/java/br/com/harbitech/school/course/Course.java

```diff
@@ -177,4 +176,16 @@ public String toString() {
                 ", subCategory=" + subcategory +
                 '}';
     }
+
+    public void  update(CourseFormUpdate categoryFormUpdate) {
+        this.name = categoryFormUpdate.getName();
+        this.codeUrl = categoryFormUpdate.getCodeUrl();
+        this.description = categoryFormUpdate.getDescription();
+        this.completionTimeInHours = categoryFormUpdate.getCompletionTimeInHours();
+        this.visibility = categoryFormUpdate.getVisibility();
+        this.targetAudience = categoryFormUpdate.getTargetAudience();
+        this.instructor = categoryFormUpdate.getInstructor();
+        this.description = categoryFormUpdate.getDevelopedSkills();
```

Comment:
Só um pequeno ajuste :)
```suggestion
        this.developedSkills = categoryFormUpdate.getDevelopedSkills();
```

Reply:
opa verdade, obrigada copy paste :-1: